internal/cmd/commands/database/init.go

```diff
@@ -8,10 +8,13 @@ import (
 	"github.com/hashicorp/boundary/internal/cmd/config"
 	"github.com/hashicorp/boundary/internal/errors"
 	"github.com/hashicorp/boundary/internal/types/scope"
+	kms_plugin_assets "github.com/hashicorp/boundary/plugins/kms"
```

Comment:
Reviewers: Same changes as in base.go. Leaving a comment here mostly for me so that if there were concerns in base.go I remember to address them here too.